ext/drbbdup/drbbdup.c

```diff
@@ -597,6 +661,21 @@ drbbdup_next_end(instr_t *instr)
     return instr;
 }
 
+/* Iterates forward to the end of the next bb copy.  If the end instruction has
+ * emulation labels, steps back to point at the first of those: i.e., this is the
+ * start of the end sequence.  Returns NULL upon failure.
+ */
+static instr_t *
+drbbdup_next_end_initial(instr_t *instr)
+{
+    while (instr != NULL && !drbbdup_is_at_end(instr))
+        instr = instr_get_next(instr);
+    if (drbbdup_is_our_emulation_marker(instr_get_prev(instr)))
+        instr = instr_get_prev(instr);
```

Comment:
Assert that this is an emu start label instr.

Reply:
Done.